app/views/trainees/edit.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render PageTitle::View.new(title: "trainees.edit", has_errors: @trainee.errors.present?) %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: "Back",
+    href: trainees_path
```

Comment:
I think this page will eventually be accessed from the `/review-draft` or `/check-details` pages. In that case, this should probably be a `DynamicBackLink` component in preparation.